tests/test_api.py

```diff
@@ -72,3 +74,28 @@ def test_api(self):
 
         t1.database.close()
         t2.database.close()
+
+    @unittest.mock.patch("sys.stdout", new_callable=io.StringIO)
+    def test_api_print(self, mock_stdout):
+        expected = "5 rows in table A\n4 rows in table B\n1 rows exclusive to table A (not present in B)\n0 rows exclusive to table B (not present in A)\n0 rows updated\n4 rows unchanged\n20.00% difference score\n\nExtra-Info:\n  rows_downloaded = 5\n"
+        t1 = connect_to_table(TEST_MYSQL_CONN_STRING, "test_api")
```

Comment:
am I missing a step with these? I assumed the setup methods would apply

Reply:
![Screen Shot 2022-11-17 at 3 08 57 PM](https://user-images.githubusercontent.com/11282254/202570571-bd36e39f-45de-4867-8791-d37d6a9534c9.png)